src/attributeFields.js

```diff
@@ -45,6 +51,15 @@ module.exports = function (Model, options = {}) {
 
     }
 
+    // set integer and string primaryKey attriubte types to GraphQLID
+    if ( attribute.primaryKey === true
+      && ( memo[key].type === GraphQLInt
+        || memo[key].type === GraphQLString
```

Comment:
type doesn't really matter i suppose

Reply:
not sure if this should be behind an option, also remember to add a test

Reply:
option: good idea

"remember"? you mean learn from scratch lol